pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -861,7 +869,27 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		}
 	}
 
-	return roles
+	// TODO roundtrip roles to pick up defaulting for API groups.  Without this, the covers check in reconcile-cluster-roles will fail.
+	// we can remove this again once everything gets group qualified and we have unit tests enforcing that.  other pulls are in
+	// progress to do that.
+	versionedRoles := []authorizationapiv1.ClusterRole{}
```

Comment:
@liggitt Removed the merge tag.  I had to add this.  Comment is above.  its temporary, but significant enough to deserve a second look.